src/app/qgisapp.cpp

```diff
@@ -4490,6 +4490,15 @@ void QgisApp::about()
     versionString += "</tr><tr><td colspan=4>" + tr( "This copy of QGIS writes debugging output." ) + "</td>";
 #endif
 
+#ifdef WITH_BINDINGS
+    if ( mPythonUtils && mPythonUtils->isEnabled() )
+    {
+      versionString += "</tr><tr><td>" + tr( "Active python plugins" ) + "</td><td colspan=3>";
```

Comment:
QLatin1String?

Reply:
Code behaves like the other pre existing string appends in this function. Should all of those be moved to qlatin1string?

Reply:
Oh, I've only seen the one at line 5002 which happened to have qlatin1string already.

Yes, I think we should almost always have QLatin1String / QStringLiteral, so if you can also change them, perfect.

Reply:
Will do.